test/parallel/test-global-console-exists.js

```diff
@@ -9,6 +9,18 @@ const EventEmitter = require('events');
 const leak_warning = /EventEmitter memory leak detected\. 2 hello listeners/;
 
 var write_calls = 0;
+
+process.on('warn', (warning) => {
+  // this will be called after the default internal
+  // process warning handler is called. the default
+  // process warning writes to the console, which will
+  // invoke the monkeypatched process.stderr.write
+  // below.
+  assert.equal(write_calls, 1);
```

Comment:
ah! right, I didn't finish updating the test. Refactoring how the warning is printed eliminated the second write.